### ISSUE TITLE
add support for rdb instance metrics

### ISSUE THREAD

**Issue by remyleone** [scaleway/scaleway-cli#1234]
## Description

Blocked by #1172 

## Version

Version    2.0.0-beta.4&#43;dev
BuildDate  unknown
GoVersion  go1.14.4
GitBranch  unknown
GitCommit  unknown
GoArch     amd64
GoOS       darwin

**Comment by jerome-quere** [comment]
Duplicate of #1161
